simple/stats/embeddings.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2023 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+import pandas as pd
+from sentence_transformers import SentenceTransformer
+from stats.data import StatVar
+from util.filehandler import FileHandler
+from util.filehandler import LocalFileHandler
+
+_MODEL_NAME = "all-MiniLM-L6-v2"
+_DCID_COL = "dcid"
+_SENTENCE_COL = "sentence"
+
+
+def build(svs: list[StatVar],
+          embeddings_fh: FileHandler | None = None) -> pd.DataFrame:
```

Comment:
super nit: since only the test needs to pass None, don't default?

Reply:
Done.